index.html

```diff
@@ -3,20 +3,11 @@
 <html>
 <head>
 	<title>ES6 Classes</title>
-	<script src="./dist/canvanoid.bundle.js"></script> 
+	
 </head>
 <body>
 	<canvas id="canvas"></canvas>
-	<script> //"babel src -d dist",
-		window.requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame || 
-									window.msRequestAnimationFrame || window.mozRequestAnimationFrame;
-					
-		var canvas = document.getElementById('canvas')
-		canvas.width = window.innerWidth - 100; 
-		canvas.height = window.innerHeight - 100; 
 
-		var game = new Canvanoid(canvas); 	
-		game.start();
-	</script>
+	<script src="./dist/canvanoid.bundle.js"></script> 
```

Comment:
The hash for browser caching is just for development, and I didn't used that in a long time, becasue I already have the checkbox for disabling cache in Chrome Dev Tools.
Still the modifications that the index.html file could receive are minimal, still not convinced of using a whole new dependency for this.

Maybe I'm just tired of over-dependency :P